db/migrate/20181108195945_migrate_conditions_tow_hat_to_resource_type.rb

```diff
@@ -0,0 +1,21 @@
+class MigrateConditionsTowHatToResourceType < ActiveRecord::Migration[5.0]
+  class Condition < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
```

Comment:
same here , it is not needed https://github.com/ManageIQ/manageiq-schema/issues/4#issuecomment-310643185